packages/ui/src/theme/tokens/components/button.ts

```diff
@@ -133,8 +160,160 @@ export const button: Required<ButtonTokens<'default'>> = {
     backgroundColor: { value: 'transparent' },
     borderColor: { value: '{colors.border.tertiary.value}' },
   },
-
   // variations
+  outlined: {
```

Comment:
Even though our default Button variation doesn't have a name, we need something to construct the custom properties pattern for the default variation that doesn't clash with the existing warning variation. 

If we did: 
```
button: { 
  info{}, 
  warning{}, 
  // ..etc 
}
```
instead, where the default button variation's color themes sit at the root, we would end up with tokens for the warning color theme that clashes with the tokens for the existing warning variation `--amplify-components-button-warning-background-color` so that is why they are nested inside of outlined here, even though there is no explicit 'outlined' variation.